.github/workflows/test.yml

```diff
@@ -13,6 +13,12 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version: 18.x
+      - run: env
```

Comment:
Was this just for testing?

Reply:
Yeah my bad.

I'm mobile so can't use the suggest feature it seems